api/authenticator.py

```diff
@@ -101,15 +101,6 @@ def create_token(self, provider_name, provider_token):
         )
         return jwt.encode(payload, self.secret_key, algorithm='HS256')
 
-    def get_credential_from_header(self, header):
```

Comment:
This method would still be useful for avoiding the copy-and-paste lookup of 'password'.